### PR TITLE
Adds Faelantern

### PR DESCRIPTION
**Pull request by vampirebat74**
## About The Pull Request

What it says on the tin. A meat lantern aberration that's designed to be even more of  pain in the rear, beloved by all. Features both a suit and E.G.O weapon of a higher risk level, in exchange for being more annoying to manage.

Sprites by Mel/Meltaculo

## Why It's Good For The Game

The Limbus Company Pokedex must be completed

## Changelog
:cl:
add: Adds Faelantern
soundadd: adds the faelantern sound folder
imageadd: adds relevant sprites for faelantern
/:cl:

### CODE REVIEW

File: code/datums/abnormality/_ego_datum/he.dm [hunk 1]
```diff
@@ -442,3 +442,12 @@
 /datum/ego_datum/weapon/nixie
 	item_path = /obj/item/ego_weapon/nixie
 	cost = 35
+
```
Comment (Coxswain-Navigator) [inline code/datums/abnormality/_ego_datum/he.dm:445]: This is a duplicate datum get rid of it.

File: code/modules/mob/living/simple_animal/abnormality/teth/faelantern.dm [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+	. = ..()
+	for(var/mob/living/carbon/human/H in lured_list)
+		EndEnchant(H)
+	icon_state = icon_dead
+	playsound(src, 'sound/abnormalities/doomsdaycalendar/Limbus_Dead_Generic.ogg', 100, 1)
+	animate(src, alpha = 0, time = 10 SECONDS)
+	QDEL_IN(src, 10 SECONDS)
+
+/mob/living/simple_animal/hostile/abnormality/faelantern/PostWorkEffect(mob/living/carbon/human/user, work_type, pe, work_time, canceled)
+	if(work_type == ABNORMALITY_WORK_REPRESSION && work_time < 16 SECONDS)
```
Comment (Coxswain-Navigator) [inline code/modules/mob/living/simple_animal/abnormality/teth/faelantern.dm:71]: Do not use work time, check temperance bonus instead. Work time is inconsistant due to server ticks.

**Comment by Litberries** [comment]
this is a fucking midwinter nightmare what the hell heavenly

**Review by Coxswain-Navigator — APPROVED** [review]
Tested pretty thoroughly. Could use an update for the attack animation but that can come later.

**Pull request merged**